clang/test/Driver/sycl-preprocess.cpp

```diff
@@ -6,7 +6,7 @@
 // RUN: %clang_cl -fsycl -P -Fi%t_output.ii %s -### 2>&1 \
 // RUN:  | FileCheck -check-prefix PREPROC_ONLY %s
 // PREPROC_ONLY: clang{{.*}} "-fsycl-is-device"{{.*}} "-E"{{.*}} "-o" "[[DEVICE_OUT:.+\.ii]]"
-// PREPROC_ONLY: clang{{.*}} "-fsycl-is-device"{{.*}} "-fsycl-int-header=[[INTHEADER:.+\.h]]"{{.*}} "-fsyntax-only"
+// PREPROC_ONLY: clang{{.*}} "-fsycl-is-device"{{.*}} "-fsycl-int-header=[[INTHEADER:.+\.h]]" "-fsycl-int-footer=[[INTFOOTER:.+\.h]]"{{.*}} "-fsyntax-only"
```

Comment:
There is no use of `INTFOOTER` in the test.  Looks like there is a disconnect in the use-footer driver support when generating and consuming preprocessed files which needs to be addressed.  When generating preprocessed files for the int-header, we generate and include the int-header.  We should also append and preprocess the int-footer when generating preprocessed files.

Reply:
Fix for preprocessing issue as mentioned above is in progress here: https://github.com/intel/llvm/pull/3878